scripts/in_container/verify_providers.py

```diff
@@ -141,6 +141,11 @@ class ProviderPackageDetails(NamedTuple):
         "adheres to: 'pyarrow<5.1.0,>=5.0.0; extra == \"pandas\"'",
         "snowflake",
     ),
+    (
+        "You have an incompatible version of 'pyarrow' installed (6.0.1), please install a version that "
+        "adheres to: 'pyarrow<8.1.0,>=8.0.0; extra == \"pandas\"'",
```

Comment:
Wowzers that's a narrow version of pyarrow they want to work with.

Reply:
I gess beam is stopping us :)

Reply:
Beam but not only:
* beam <8.0.0
* databricks-sql-connector < 8.0.0
* pandas-gbq <9.0.0dev <- not holiding us back
* sqlalchemy-bigquery <7.0.0

Reply:
At least this is what pipdeptree shows us in our image (there might be others that do it transitively via extras and those are only possible to see by inspecting individual packages)